service/routes.py

```diff
@@ -89,6 +90,126 @@ def create_wishlists():
         status.HTTP_201_CREATED
     )
 
+######################################################################
+# RETRIEVE A WISHLIST
+######################################################################
+@app.route("/wishlists/<int:wishlist_id>", methods=["GET"])
+def get_wishlist(wishlist_id):
+    """
+    Retrieve a single wishlist
+    This endpoint will return a wishlist based on it"s id
+    """
+    app.logger.info("Request for wishlist with id: %s", wishlist_id)
+    wishlist = Wishlist.find_by_id(wishlist_id)
+    if not wishlist:
+        return(
+            f"Wishlist with id {wishlist_id} was not found",
+            status.HTTP_404_NOT_FOUND
+        )
+    return make_response(jsonify(wishlist.serialize()), status.HTTP_200_OK)
```

Comment:
Would it be better to combine `get_wishlist` with `list_products_in_wishlist`? That way when you want to see a particular wishlist you don't have to make two calls to see the id, name and what products are in it. Or am I missing something?